practices/refactoring/readme.md

```diff
@@ -0,0 +1,93 @@
+# Refactoring
+
+Refactoring is very tightly coupled to Simple Design and TDD. TDD enables simple design, while the practice of refactoring ensures that our code bases adapt to new or changing requirements while still keeping its maintainability.
+
+## The What and The Why
+
+### The What
+
+Refactoring is a systematic process for improving code without creating new functionality.
+
+### The Why
+
+The goal of refactoring is to remediate technical debt and transform sloppy code into [clean code](/corevalues/cleancode/cleancode.md) with a simple design.
+
+#### Technical Debt and Causes
+
+Technical debt reflects the implied cost of reworking code caused by choosing a quick and easy solution instead of a better more involved approach. Technical debt is one of the main reasons you would want to refactor code, with the main culprits being:
+
+1. Business pressure
```

Comment:
this list isn't actually ordered, and the extra spaces between lines are confusing. Some interpreters will treat them each as their own list which can be problematic because

```
1. interestingly
1. the numbers for an ordered list
1. don't impact the rendering
```
results in:

1. interestingly
1. the numbers for an ordered list
1. don't impact the rendering

Reply:
Changed to a bulleted list. No need for ordered anyways.